admin/admin_login.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+// Admin login credentials
+$admin_username = "admin";
```

Comment:
Hard codded admin username and password, which might differ from admin to admin.